lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -77,43 +77,35 @@ class WP_Theme_JSON_Resolver_Gutenberg {
 	 */
 	protected static $user_custom_post_type_id = null;
 
-	/**
-	 * Container to keep loaded i18n schema for `theme.json`.
-	 *
-	 * @since 5.8.0 As `$theme_json_i18n`.
-	 * @since 5.9.0 Renamed from `$theme_json_i18n` to `$i18n_schema`.
-	 * @var array
-	 */
-	protected static $i18n_schema = null;
-
-	/**
-	 * `theme.json` file cache.
-	 *
-	 * @since 6.1.0
-	 * @var array
-	 */
-	protected static $theme_json_file_cache = array();
-
```

Comment:
We might need to keep them if someone is extending the class and expects these to exist.

Reply:
I don't think that matters here as the class is marked private. It must not be overwritten for this purpose, and if anyone does so it's at their own risk (similar to e.g. using "experimental" APIs in Gutenberg).